### DISCUSSION TITLE
Contribution to your project

### DISCUSSION THREAD

**Discussion by aliferouss19** [acheong08/ChatGPT#91] [Ideas]
Hi, 

I want to contribute to your project. I can make translation, code, 2D graphics - icons, logos. Will you be interested?

Thank you in advance.

Valeriia Zhuravska

**Comment by acheong08** [comment]
I am most definitely interested. 

The main issue with this project is spaghetti code. If you could review a bit of it and refactor to optimize, it would be appreciated as well. I want this project to be maintainable by others in the future so some conventions such as PyLint would be nice. 

In the future: Having multiple translations of the readme would make this project much more accessible. A logo would also be cool.

Sincere thanks,
Antonio

**Reply by aliferouss19** [reply]
Unfortunately I don't really know how to write the code, but I will be glad to suggest a new logo :) I will work on it, thank you.

**Reply by acheong08** [reply]
A logo will work just as well. Any contribution is welcome

**Reply by aliferouss19** [reply]
![logo-02](https://user-images.githubusercontent.com/119953057/206899875-36cf26db-934c-4cea-bb8a-57ebca44ab26.png)
Hi, so here's my logo suggestion

**Reply by acheong08** [reply]
Thank you! I will use it
